report_thesis/src/sections/background/linear_and_regularization_models/rigde_regression.tex

```diff
@@ -0,0 +1,21 @@
+\subsubsection{Ridge Regression}
+A limitation of \gls{ols} regression is that it cannot handle multicollinearity effectively.
```

Comment:
I haven't been able to find anything on the multicollinearity claims we make in the source. Can you give me the quotes where it's said? For all these sections.

Reply:
Multicollinearity occurs when two or more predictor variables in a multiple regression model are highly linearly related. This means that one predictor variable can be linearly predicted from the others with a substantial degree of accuracy.

This is not the same as there being a collinearity (pairwise) between predictor variables.

Reply:
I found a source that explictly states it - here is an excerpt from the book:
![image](https://github.com/chhoumann/thesis-chemcam/assets/57709490/2d0008e7-8d34-4b68-9440-aa505440c731)

This paper also cites this book as their source for a similar claim:
https://philstat.org/index.php/MSEA/article/download/1238/794

![image](https://github.com/chhoumann/thesis-chemcam/assets/57709490/8904c8b6-08af-485d-beae-af6ac229135c)
![image](https://github.com/chhoumann/thesis-chemcam/assets/57709490/ab6c0413-282e-4c22-a332-7fd33ebe0265)

Reply:
I'm not a fan of retroactively finding sources to back up claims. You made the claim for a reason at first, which I suppose was based on James? So if the claim is in that book, why not just use that?

Especially now: what about all the other claims you make about multicollinearity?

> A limitation of \gls{ols} regression is that it cannot handle multicollinearity effectively.

> Despite \gls{lasso} regression's ability to perform feature selection and its effectiveness in settings with multicollinearity, it can struggle when the dataset contains globally highly correlated features.

> \gls{lasso} regression addresses the same issue of multicollinearity as ridge regression but does so by introducing a different regularization term to the \gls{ols} regression objective function.

None of these are covered by the sources you provide.

And it isn't immediately clear to me (could be skill issue, sure) why Ridge handles multicollinearity. It _seems_ to be true, but it isn't clear why.